runtime/Java/src/org/antlr/v4/runtime/atn/ATNSerializer.java

```diff
@@ -314,6 +314,11 @@ public void serializeCodePoint(IntegerList data, int cp) {
 					data.add(actionIndex != -1 ? actionIndex : 0xFFFF);
 					break;
 
+				case LESS:
```

Comment:
That's a problem. You should add a new UUID to enable the LESS lexer mode, just like it had been done with the lexer actions. Otherwise there can be conflicts with generated parser/lexer files, using a runtime that doesn't support the LESS command. We could require that their versions always match, but traditionally this kind of counter measure has been used and we should continue with that approach.